openai/tests/test_endpoints.py

```diff
@@ -28,3 +28,17 @@ def test_completions_multiple_prompts():
         prompt=["This was a test", "This was another test"], n=5, engine="ada"
     )
     assert len(result.choices) == 10
+
+
+def test_completions_model():
+    result = openai.Completion.create(prompt="This was a test", n=5, model="ada")
+    assert len(result.choices) == 5
+    assert result.model.startswith("ada:")
+
+
+def test_completions_headers():
+    result = openai.Completion.create(
+        prompt="This was a test", n=5, model="bogus", headers={"Openai-Model": "ada"}
```

Comment:
Do we want to make this visible to the world? 

It is accepted but I don't think we want to incentivize it